kolibri/plugins/facility/assets/src/views/users/UsersRootPage/index.vue

```diff
@@ -56,7 +56,7 @@
           <template #userActions>
             <component
               :is="canAssignCoaches ? 'router-link' : 'span'"
-              :to="overrideRoute($route, { name: PageNames.ASSIGN_COACHES_SIDE_PANEL })"
+              :to="overrideRoute($route, { name: PageNames.SSAIGN_COACHES_SIDE_PANEL })"
```

Comment:
Seems like an unnoticed change here is breaking the open assign coaches side panel